onvm/onvm_nflib/onvm_nflib.c

```diff
@@ -405,34 +417,32 @@ onvm_nflib_run_callback(struct onvm_nf_context *nf_context, pkt_handler_func han
 void *
 onvm_nflib_thread_main_loop(void *arg) {
         struct rte_mbuf *pkts[PACKET_READ_SIZE];
-        struct onvm_nf_context *nf_context;
+        struct onvm_nf_local_ctx *nf_local_ctx;
         struct onvm_nf *nf;
         uint16_t nb_pkts_added;
-        struct onvm_nf_info *info;
         pkt_handler_func handler;
-        callback_handler_func callback;
+        callback_func callback;
```

Comment:
`callback_func` is ambiguous; it is not clear what it is "calling back".

Reply:
Agreed, not sure how to name it though. Its a main loop callback, called in every iteration, ideas?

Reply:
nf_processing_callback?
nf_custom_callback?

Reply:
Does the name really need "callback"?  Removing that word will make naming easier.

Also, it would be good to do the same for `pkt_handler_func handler`.  Name it something like `pkt_handler_func process_packet`

Reply:
Hmm like `nf_custom_processing_func custom_processing`  ?